client/src/components/TextArea/Text.stories.js

```diff
@@ -1,7 +0,0 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
-
-import Text from './Text';
-
-storiesOf('Text', module)
-    .add('Default', () => <Text pClassList='text-area-title' textAreaClassList='text-area'/>)
```

Comment:
Did you mean to delete this file? If you merge this branch into develop and have deleted this file it will remove it from the delete branch.

Reply:
This was Dilmourads.